### ISSUE TITLE
RST: Standardized Age Groups does not check for duplicate groups RST

### ISSUE THREAD

**Issue by ryanzomorrodi** [CEHI-code-repos/RateStabilizingTool2#15]
When duplicate groups are used:
```
﻿Traceback (most recent call last):
﻿  File "<string>", line 394, in execute
﻿  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\numpy\core\records.py", line 730, in fromrecords
﻿    return fromarrays(arrlist, formats=formats, shape=shape, names=names,
﻿           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
﻿  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\numpy\core\records.py", line 655, in fromarrays
﻿    descr = format_parser(formats, names, titles, aligned, byteorder).dtype
﻿            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
﻿  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\numpy\core\records.py", line 149, in __init__
﻿    self._setfieldnames(names, titles)
﻿  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\numpy\core\records.py", line 198, in _setfieldnames
﻿    raise ValueError("Duplicate field names: %s" % _dup)
﻿ValueError: Duplicate field names: ['median_35to64', 'maxCI_35to64', 'reliable_35to64']
```

This is especially annoying because it happens after the model is completed. Best course of action is probably just to stop the user from specifying duplicate groups

**Comment by ryanzomorrodi** [comment]
Turns out there is a check, it just doesn't work

**Comment by ryanzomorrodi** [comment]
Fixed in [3baf4bd](https://github.com/CEHI-code-repos/RateStabilizingTool2/commit/3baf4bda9df516d1f38786b53a429c87b2f18405)
